ts/Extensions/Sonification/ChartSonify.ts

```diff
@@ -825,8 +828,8 @@ namespace ChartSonify {
     }
 
     export interface ChartSonificationStateObject {
-        currentlyPlayingPoint?: Highcharts.SonifyablePoint;
-        timeline?: Highcharts.Timeline;
+        currentlyPlayingPoint?: Sonification.SonifyablePoint;
```

Comment:
`Sonification.SonifyablePoint` should be integrated into `PointSonify.Composition`. Then you can delete the custom `this` scope of the `PointSonify.Composition` functions.